package.json

```diff
@@ -78,6 +78,7 @@
     "@types/jquery": "^3.3.31",
     "@types/node": "^10.14.19",
     "codelyzer": "^5.0.1",
+    "cypress": "^3.4.1",
```

Comment:
Not the right Cypress version. See #916 that i pinged you on.

Reply:
Ah shoot, forgot about that. I thought that doing a rebase on develop then a force push would bring everything up to date.

Reply:
Fixed.